.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -138,19 +138,19 @@ def render(self):
 
 WORKFLOW_DATA = [
     # VS2019 CUDA-10.1
-    WindowsJob(None, _VC2019, CudaVersion(10, 1)),
-    WindowsJob(1, _VC2019, CudaVersion(10, 1)),
-    WindowsJob(2, _VC2019, CudaVersion(10, 1)),
+    WindowsJob(None, _VC2019, CudaVersion(10, 1), master_only=True),
+    WindowsJob(1, _VC2019, CudaVersion(10, 1), master_only=True),
+    WindowsJob(2, _VC2019, CudaVersion(10, 1), master_only=True),
     WindowsJob('_azure_multi_gpu', _VC2019, CudaVersion(10, 1), multi_gpu=True, nightly_only=True),
     # VS2019 CUDA-11.1
     WindowsJob(None, _VC2019, CudaVersion(11, 1)),
     WindowsJob(1, _VC2019, CudaVersion(11, 1), master_only=True),
     WindowsJob(2, _VC2019, CudaVersion(11, 1), master_only=True),
     # VS2019 CPU-only
     WindowsJob(None, _VC2019, None),
-    WindowsJob(1, _VC2019, None, master_only=True),
-    WindowsJob(2, _VC2019, None, master_only=True),
-    WindowsJob(1, _VC2019, CudaVersion(10, 1), force_on_cpu=True, master_only=True),
+    WindowsJob(1, _VC2019, None),
+    WindowsJob(2, _VC2019, None),
+    WindowsJob(1, _VC2019, CudaVersion(10, 1), force_on_cpu=True),
```

Comment:
Are shard1 and shard2 for Windows CUDA and CPU jobs identical? If so, than perhaps we can run force_on_cpu shard1 + CPU shard2 on PR? 
```suggestion
    WindowsJob(1, _VC2019, None, master_only=True),
    WindowsJob(2, _VC2019, None),
    WindowsJob(1, _VC2019, CudaVersion(10, 1), force_on_cpu=True),
```